src/helpers/client.js

```diff
@@ -0,0 +1,28 @@
+import { config } from '../config'
+
+async function getSubjects(coordinates) {
+  try {
+    const { northEast, southWest } = coordinates
+    const minLat = southWest.lat
+    const maxLat = northEast.lat
+    const minLon = southWest.lng
+    const maxLon = northEast.lng
+    const endpoint = `${config.dbEndpoint}/subjects?minLat=${minLat}&minLon=${minLon}&maxLat=${maxLat}&maxLon=${maxLon}`
+    const response = await fetch(endpoint)
+      .then(response => {
+        if (response.status === 500) {
+          return { statusCode: 500, title: 'Error connecting to database' }
+        }
+        if (response.status === 422) {
+          return { statusCode: 422, title: 'Missing one of parameters: minLat, maxLat, minLon, maxLon' }
+        }
+        if (response.ok) return response.json()
+      })
+      .then(data => data)
+    return response
```

Comment:
The mix of async/await and the promise syntax makes what's going on here a bit confusing, since await is already syntatic sugar for a promise so I'd clear this up with a refactor:
```suggestion
    const response = await fetch(endpoint)

    if (!response.ok) {
      throw response
    }
    
    return response.json()
```

`response.ok` covers the HTTP 200s, so any other HTTP errors or other network errors can be thrown. The catch block will catch the thrown response object and then you can do specific handling for certain HTTP error codes there if needed.